src/SFA.DAS.EmployerAccounts/Models/Commitments/CohortV2.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.EmployerAccounts.Models.Commitments
+{
+    public class CohortV2
```

Comment:
Please remove the V2.  Put the V2 in the namespace and move this to a V2 folder so it doesn't clash wit the existing Cohort.

namespace SFA.DAS.EmployerAccounts.Models.Commitments.V2

Then we don't have to name objects with names that have no meaning.